mandatory/2-function-creation.js

```diff
@@ -6,7 +6,14 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num)
+{
+    if (typeof num == "number" && num % 2 == 0 && num <= 100)
+    {
+        return true;
+    }
+    return false;
+}
```

Comment:
```suggestion
function validate(num) {
    return typeof num == "number" && num % 2 == 0 && num <= 100)
}
```

Reply:
Hi Kerim, your code has been pretty well reviewed already. I can't really add anything on top of what Dom and Saadia have already suggested. Nice work!